### PR TITLE
Temporarily disable some integration tests to unblock queue

### PR DESCRIPTION
**Pull request by spadgett**
Run the integration tests, but don't fail if only the integration tests fail. See #2200

Merge will still fail on dist differences or unit test failures.

### CODE REVIEW

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
@jeff-phillips-18 @benjaminapetersen Any objection to this? Our merge queue has been broken for several days now. We haven't been able to find a simple fix.

I think the right path forward is to update Protractor in #2083, which we need to do anyway.

File: Makefile [hunk 1]
```diff
@@ -44,5 +44,5 @@ build: install
 test: build
 	hack/verify-dist.sh
 	hack/test-headless.sh test $(GRUNT_FLAGS)
-	hack/test-headless.sh test-integration $(GRUNT_FLAGS)
+	-hack/test-headless.sh test-integration $(GRUNT_FLAGS)
```
Comment (benjaminapetersen) [inline Makefile]: only thought, comment this?  the `-` runs it but wont fail?

**Comment by spadgett** [comment]
[merge]

**Pull request merged**